recipes/ffmpeg/all/conanfile.py

```diff
@@ -355,7 +355,7 @@ def _target_os(self):
             return target_os
 
     def _patch_sources(self):
-        if self._is_msvc and self.options.with_libx264 and not self.options["libx264"].shared:
+        if self._is_msvc and self.options.with_libx264 and not self.options["libx264"].shared and tools.Version(self.version) <= "5.0":
```

Comment:
This will block the merge :/ you  need to update the imports

```suggestion
        if self._is_msvc and self.options.with_libx264 and not self.options["libx264"].shared and Version(self.version) <= "5.0":
```